kitsune/sumo/models.py

```diff
@@ -50,3 +50,29 @@ def __init__(
         return super(LocaleField, self).__init__(
             max_length=max_length, default=default, choices=choices, *args, **kwargs
         )
+
+
+class BetweenQuerySet(models.QuerySet):
+    """
+    QuerySet adding `before(date)` and `after(date)` methods,
+    to filter for rows before or after a datetime.
+
+    The `updated_column_name` property must be defined on a model using this,
+    as that will be used as the column to filter on in the before and after methods.
+    """
+
+    def before(self, date):
```

Comment:
Why not combining these into a `range(self, start, stop)` with stop defaulting to `now` if none is provided?

Reply:
And if we go down this road, we probably don't need the manager at all. We could use directly into the management command and if we need it in more places (in the future or now) we could add a method/property in the base model.

Reply:
I'd prefer to go straight for the generic option, but yes something like `BaseModel.object_range(start, stop)` might be better - involving no fiddling around with existing managers on our models.